cf-check/utilities.h

```diff
@@ -6,8 +6,8 @@
 // These functions should be moved to libutils once cf-check is
 // implemented and backported.
 
-Seq *argv_to_seq(int argc, char **argv);
+Seq *argv_to_seq(int argc, const char *const *argv);
```

Comment:
Maybe we should at some point move this function to `libutils`? Seems useful.

Reply:
Yep. TODO.